test/e2e/gameserver_test.go

```diff
@@ -990,6 +990,42 @@ func TestGameServerSetPlayerCapacity(t *testing.T) {
 	})
 }
 
+func TestPlayerConnectWithCapacityZero(t *testing.T) {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		t.SkipNow()
+	}
+	t.Parallel()
+	ctx := context.Background()
+
+	gs := framework.DefaultGameServer(framework.Namespace)
```

Comment:
21: e2e-feature-gates(PlayerTracking=true) is failed.
https://github.com/googleforgames/agones/blob/79c79d2ad2fd05bc947cc005f6a354c96f0d3ca2/test/e2e/framework/framework.go#L160
Maybe simple-game-server:0.6 should be changed to simple-game-server:0.11 like https://github.com/googleforgames/agones/pull/2413